starters/nuxt3-pinia-vuetify/.storybook/preview.js

```diff
@@ -0,0 +1,56 @@
+import "vuetify/dist/vuetify.css";
+import { create } from '@storybook/theming';
+import { action } from '@storybook/addon-actions';
+import { app, addDecorator } from '@storybook/vue3';
+import { Vuetify } from '../plugins/vuetify';
+import { createPinia } from 'pinia';
+
+
+// Support for pina in Storybook
+const pinia = createPinia();
+app.use(pinia);
```

Comment:
I think there's a better way to setup storybook - they updated their docs a few weeks ago: https://storybook.js.org/tutorials/intro-to-storybook/vue/en/data/